### PR TITLE
Docs: add security page

### PR DESCRIPTION
**Pull request by evilnick**
Adds the security explainer and adjusts a few other pages accordingly.

### CODE REVIEW

File: docs/src/explanation/security.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+As detailed in the [snap documentation][], an application installed from a snap
+is inherently more secure than a traditionally installed application.
+Snap-based applications are installed into a sandboxed, self contained
+environment which restricts its ability to interact with the rest of user
+space.
+
+## Security of the OCI images
+
+**Canonical Kubernetes** relies on OCI standard images published as `rocks` to
+deliver the services which run and facilitate the opration of the Kubernetes
```
Comment (eaudetcobello) [inline docs/src/explanation/security.md:23]:
```suggestion
deliver the services which run and facilitate the operation of the Kubernetes
```

File: docs/src/explanation/security.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+cluster. Many of the frequently used 'standard' OCI images have known security
```
Comment (eaudetcobello) [inline docs/src/explanation/security.md:24]: Let's maybe not start the sentence with a negative about OCI images while having just talked about the fact that Rocks are OCI images. Could we flip it around and say "OCI images give Canonical a way to [...] in ways that other commonly used OCI images cannot."

File: docs/src/explanation/security.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+release of Kubernetes. For more information on how these images are maintained
+and published, see the [Rockcraft documentation][rocks-security]. 
+
+## Kubernetes Security
+
+The Kubernetes cluster deployed by Canonical Kubernetes can be secured using
+any of the methods and options described by the upstream
+[Kubernetes Security Documentation][].
+
+Canonical Kubernetes enables RBAC(Rules Based Access Control)by default.
```
Comment (eaudetcobello) [inline docs/src/explanation/security.md:37]:
```suggestion
Canonical Kubernetes enables RBAC (Rules Based Access Control) by default.
```

File: docs/src/explanation/security.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+space.
+
+## Security of the OCI images
+
+**Canonical Kubernetes** relies on OCI standard images published as `rocks` to
+deliver the services which run and facilitate the opration of the Kubernetes
+cluster. Many of the frequently used 'standard' OCI images have known security
+vulnerabilities, but the use of `Rockcraft` and `rocks` gives Canonical a way
+to maintain and patch images to remove vulnerabilities at their source, which
+is fundamental to our commitment to a sustainable Long Term Support(LTS)
```
Comment (eaudetcobello) [inline docs/src/explanation/security.md:27]:
```suggestion
is fundamental to our commitment to a sustainable Long Term Support (LTS)
```

File: docs/src/explanation/security.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+instances, anyone with credentials to the cloud where it is deployed may also
+have access to your cluster. Describing the security mechanisms of these clouds
+is out of the scope of this documentation, but you may find the following links
+useful.
+
+-   Amazon Web Services -	<https://aws.amazon.com/security/>
+-   Google Cloud Platform	- <https://cloud.google.com/security/>
+-   Metal As A Service(MAAS) -  <https://maas.io/docs/snap/3.0/ui/hardening-your-maas-installation>
+-   Microsoft Azure	- <https://docs.microsoft.com/en-us/azure/security/azure-security>
+-   VMWare VSphere	- <https://www.vmware.com/security/hardening-guides.html>
```
Comment (eaudetcobello) [inline docs/src/explanation/security.md:51]: I think the formatting is a bit off here.

**Review by eaudetcobello — COMMENTED** [review]
Nice doc! :) Left a couple of comments.

File: docs/src/explanation/security.md (hunk 5 above)
Comment (evilnick) [reply docs/src/explanation/security.md:51]: yeah, it was formatted as a table before. Will fix that, thanks

**Pull request merged**